src/form/validators.ts

```diff
@@ -120,33 +120,39 @@ export const dateBeforeValidatorGenerator =
   };
 
 export const dateAfterOrEqualValidatorGenerator =
-  (comparisonValue?: string, customError?: string) =>
-  (value?: string): string | null => {
-    if (value && comparisonValue) {
+  (customError?: string) =>
+  (value: string, values: any): string | null => {
```

Comment:
Would it make sense to type this so that at least `values.search.start_date` is expected in the type?

Reply:
Yes makes sense